src/content/getting-started/secure-your-app/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Secure your app
+description: Learn how to secure your app on the Giant Swarm platform.
+weight: 70
+last_review_date: 2024-10-08
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - How do I observe the platform metrics and logs for my application?
```

Comment:
I think this file is not needed?

Reply:
we are linking the next step, so I added it empty to don't have broken links